apps/testingaccessibility/src/utils/get-resource-paths.test.ts

```diff
@@ -0,0 +1,142 @@
+import {getPathForLesson, getPathForSection} from './get-resource-paths'
+
+test('correctly resolves lesson path', async () => {
+  const lessonSlug = 'testing-zoom-in-the-browser'
+  const expectedLessonPath = getPathForLesson(lessonSlug, modules as any)
+
+  expect(expectedLessonPath).toStrictEqual({
+    module: 'manual-testing',
+    section: 'magnification-and-zoom-testing',
+    lesson: 'testing-zoom-in-the-browser',
+  })
+})
+
+test('correctly resolves section path', async () => {
+  const sectionSlug = 'intro-and-setup'
+  const expectedSectionPath = getPathForSection(sectionSlug, modules as any)
+
+  expect(expectedSectionPath).toStrictEqual({
+    module: 'manual-testing',
+    section: 'intro-and-setup',
+  })
+})
+
+const modules = [
+  {
+    slug: 'manual-testing',
+    title: 'Manual Testing',
+    sections: [
+      {
+        lessons: null,
+        slug: 'intro-and-setup',
+        title: 'Intro & Setup',
+      },
+      {
+        lessons: [
+          {
+            slug: 'identify-issues-by-hitting-tab',
+            title: 'Identify Issues by Hitting Tab',
+          },
```

Comment:
I wasn't sure if making an actual request to sanity is a good idea given the content is behind a paywall. so I decided to hardcode response of a minimal `modules` query corresponding to what we're doing in the app.

Reply:
in this case you wouldn't need to mock the sanity request and instead you'd have data in the correct "shape" so this is great